### ISSUE TITLE
Add `Configuration.php` to resources for Container

### ISSUE THREAD

**Issue by DracoBlue** [sensiolabs/SensioGeneratorBundle#309]
Currently the generated Configuration.php wouldn't refresh the cache if it's modified in dev environment.

But if one adds

``` php
        $container->addResource(new FileResource(__DIR__.'/Configuration.php'));
```

in `DependencyInjection/*Extension.php` even modifications of the `DependencyInjection/Configuration.php` will trigger a cache refresh.

What do you think?

**Comment by javiereguiluz** [comment]
Any other comment about this proposal?

**Comment by javiereguiluz** [comment]
Closing it as "won't fix" for the reasons explained in #412.
